.github/workflows/smoke-test-prod-podman.yml

```diff
@@ -0,0 +1,108 @@
+name: Production Smoke Test Podman (via compose.prod.yml)
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Copy environment file
+        run: cp example.prod.env .prod.env
+
+      - name: Add SECRET_KEY to environment file
+        run: |
+          echo -e "\nSECRET_KEY=${{ secrets.SECRET_KEY }}" >> .prod.env
+      - name: Set up Podman
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install podman
+          pip install podman-compose
+
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+          podman --version
+          which podman-compose
+
+      - name: Start containers
+        run: podman-compose -f compose.prod.yml up -d
+
+      - name: Wait for database to be ready
+        run: |
```

Comment:
The indentation for this block is not consistent with the other blocks. There should be two spaces of indentation after `run: |` and the `until` and `if` blocks are not indented, either.